### PR TITLE
Fix: orderbook spotprice miscalculation

### PR DESCRIPTION
**Pull request by iboss-ptk**
- bump sqsdomain
- fix orderbook spot price miscalculation

### CODE REVIEW

File: router/usecase/pools/routable_cw_orderbook_pool.go [hunk 1]
```diff
@@ -155,21 +155,28 @@ func (r *routableOrderbookPoolImpl) SetTokenOutDenom(tokenOutDenom string) {
...
 	tick := r.OrderbookData.Ticks[tickIdx]
 
 	// Calculate the price for the current tick
-	return clmath.TickToPrice(tick.TickId)
+	tickPrice, err := clmath.TickToPrice(tick.TickId)
+	if err != nil {
+		return osmomath.BigDec{}, err
+	}
+
+	return convertValue(osmomath.OneBigDec(), tickPrice, directionOut), nil
```
Comment (p0mvn) [inline router/usecase/pools/routable_cw_orderbook_pool.go]: Can be precomputed since `converValue` does not use mutative math

**Review by p0mvn — APPROVED** [review]
Nice catch

File: router/usecase/pools/routable_cw_orderbook_pool.go (hunk 1 above)
Comment (iboss-ptk) [reply router/usecase/pools/routable_cw_orderbook_pool.go]: I'm not sure what you mean. Could you elaborate?

Comment (p0mvn) [reply router/usecase/pools/routable_cw_orderbook_pool.go]: Define `var oneDec = osmomath.OneBigDec()` in global space and reference here

**Pull request merged**